index.html

```diff
@@ -227,34 +227,154 @@ <h2 class="q-card__title text-secondary">Answer</h2>
+            <li class="tag-list__item">dolor</li>
+          </ul>
+
+          <div class="bookmark">
+            <input checked class="hidden" type="checkbox" id="q1bmb" />
+            <label class="bookmark__btn" for="q1b">
+              <i class="bookmark__btn--empty far fa-bookmark"></i>
+              <i class="bookmark__btn--filled fas fa-bookmark"></i>
+            </label>
+          </div>
+        </li>
+      </ul>
+    </main>
+    <!-- CREATE-CONTENT  -->
+    <main id="CREATE" class="js-content xcentered wv-full hidden">
+      <span class="sr-only" id="form-label">Create a new Question</span>
+      <form aria-labelledby="form-label" action="" class="submit-form card">
+        <label class="text-lg text-primary" for="question-field"
+          >Question</label
+        >
+        <textarea
+          class="submit-form__input submit-form__input--question"
+          name="question"
+          id="question-field"
+          placeholder="Write your question here..."
+        ></textarea>
+        <label class="text-lg text-secondary" for="answer-field">Answer</label>
+        <textarea
+          class="submit-form__input submit-form__input--answer"
+          name="answer"
+          id="answer-field"
+          placeholder="Write your answer here..."
+        ></textarea>
+        <label class="text-lg text-dark" for="tags-field">Tags</label>
+        <input
+          type="text"
+          class="submit-form__input submit-form__input--tags"
+          name="tags"
+          id="tags-field"
+          placeholder="tags..."
+        />
+        <button class="submit-form__btn bg-gradient-secondary">
+          Submit Question
+        </button>
+      </form>
+    </main>
+    <!-- PROFILE-CONTENT  -->
+    <main id="PROFILE" class="js-content xcentered hidden">
+      <div class="card profile-card">
+        <img
+          class="profile-card__img"
+          src="https://source.unsplash.com/random/300x300?portrait"
+          alt=""
+          height="200"
+          width="200"
+        />
+        <h2 class="profile-card__name">John Doe</h2>
+        <p class="profile-card__description">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Incidunt
+          adipisci laborum voluptatum, quo ipsa voluptates temporibus ullam
+          mollitia cupiditate illum odit at velit, nobis suscipit nam!
+          Praesentium, accusamus sed maiores veritatis nulla consequatur,
+          repellendus ex, recusandae expedita quod facere nisi.
+        </p>
+        <h3 class="profile-card__skill-header">Skills</h3>
+        <ul class="tag-list">
+          <li class="tag-list__item">HTML</li>
+          <li class="tag-list__item">CSS</li>
+          <li class="tag-list__item">React</li>
+          <li class="tag-list__item">Design</li>
+        </ul>
+        <a class="profile-card__logout-btn" href="#tologoutpage">logout</a>
+      </div>
+    </main>
 
     <nav class="zz-top">
       <dl class="menu">
```

Comment:
Nice idea to use a dl!
I am wondering for days now how to create a useful nav without to many elements like ul and li between nav and the actual buttons. Do you know whether this is best practice? Would help me a lot!

Reply:
I guess not, lets ask Jerry!